libs/@guardian/libs/src/logger/subscriptions.ts

```diff
@@ -47,9 +47,13 @@ const teamStyles = {
 		background: '#C74600',
 		font: '#FEF9F5',
 	},
+	perf: {
+		background: '#FFD700',
+		font: '#000000',
+	},
 } as const;
```

Comment:
What about a little `satisfies`, here?

```suggestion
} as const satisfies Record<string, { background: string; font: string}>;
```